src/components/Card/Card.jsx

```diff
@@ -110,7 +107,6 @@ const EmptyMessageWrapper = props => {
 CardWrapper.propTypes = {
   children: PropTypes.node.isRequired,
   dimensions: PropTypes.shape({ x: PropTypes.number, y: PropTypes.number }).isRequired,
-  showOverflow: CardPropTypes.showOverflow.isRequired,
```

Comment:
[Removal of a prop should result in a breaking change major version bump.](https://github.com/IBM/carbon-addons-iot-react/blob/master/docs/guides/versioning.md#an-existing-prop-is-removed) 

In this case you state there's no functionality being removed but removing the prop could cause application test automation failures. We can [deprecate the prop instead](https://github.com/IBM/carbon-addons-iot-react/blob/master/docs/guides/versioning.md#an-existing-prop-is-deprecated). 

The ``${iotPrefix}--card--wrapper--overflowing`` className is also being removed, which consumers could be targeting. Could we add a `data-testid` to the outermost element and provide it in a deprecation notice? see below:



```
import deprecate from '../../internal/deprecate';

...

showOverflow: deprecate(
  PropTypes.oneOfType([
    PropTypes.shape({
      type: PropTypes.oneOf([Tabs]),
    }),
  ]),
  '\nThe prop `showOverflow` for PageTitleBar has been deprecated. Instead, use the default implementation of the tooltip. The `${iotPrefix}--card--wrapper--overflowing` class has been removed, target `data-testid="card-cardwrapper"` instead.'
),
```

The deprecation message could probably be improved, but you get the idea.

Reply:
Makes total sense, thanks for explaining.

Reply:
Fixed